src/clr-core/internal/utils/dom.ts

```diff
@@ -17,3 +19,39 @@ export function getElementWidthUnless(element: HTMLElement, unless: boolean) {
   }
   return '';
 }
+
+export function isHTMLElement(el: any) {
+  if (el) {
+    return el instanceof HTMLElement;
+  }
+  return false;
+}
+
+export function addAttributeValue(element: HTMLElement, attr: string, value: string) {
+  if (element) {
```

Comment:
It looks like this is appending an attributeValue. so if I have an element like '<div title="ohai">` and I pass in....

```
addAttributeValue(thatDiv, 'title', 'kthxbye')
```
...the result would be...
```
<div title="ohai kthxbye">
```
...instead of...
```
<div title="kthxbye">
```
But most HTML attrs aren't like that. This would seem to be most useful for adding classnames or (in the case of our layouts) adding layout designations.

Is this intentional?

Reply:
Yup, this was intentional. As you correctly pointed out:

> This would seem to be most useful for adding classnames or (in the case of our layouts) adding layout designations.

This function is used to add and remove the `no-scrolling` by changing the `cds-layout` attr on body. An earlier suggestion in the review was to leverage our `cds-layout` in our focus trap base class to achieve this. We shouldn't remove other values that are set by the application as it pertains to our layout, and this function facilitates that by only adding or removing the value that's passed to this function. That's why the function name is called `addAttributeValue` as opposed to `setAttributeValue` (besides, this functionality is already available natively and wouldn't need this function).  

Do you have a suggestion to make this point clearer? Maybe a better name? Constrain this to layout attr? Or do you disagree that we should be using `cds-layout` attribute for this, and suggest we revert to using classnames?

Reply:
maybe `appendAttributeValue`? although I see no issues with `addAttributeValue` tbh. i just wanted to be sure i understood the intention. makes total sense in the case you describe. i don't think there's a need to change the name.